gensim/models/word2vec.py

```diff
@@ -1181,33 +1189,103 @@ def intersect_word2vec_format(self, fname, lockf=0.0, binary=False, encoding='ut
         logger.info("merged %d vectors into %s matrix from %s" % (overlap_count, self.wv.syn0.shape, fname))
 
     def most_similar(self, positive=[], negative=[], topn=10, restrict_vocab=None, indexer=None):
+        """
+        Please refer to the documentation for
+        `gensim.models.KeyedVectors.most_similar`
+        This is just a forwarding function.
+        In the future please use the `gensim.models.KeyedVectors` instance in wv
```

Comment:
It's a question for @tmylk , but I think we want to nudge users toward using the new API, yes.

The line breaks definitely not good, hurts readability.